### ISSUE TITLE
hifiasm's output with Hi-C Integrated Assembly model works badly in juicer and 3ddna pipeline

### ISSUE THREAD

**Issue by qukunjing** [chhylp123/hifiasm#486]
Hi,
I have a male heterozygous genome and its heterozygosity is relatively high, Genomescope is estimated to be ~4%, I divided it into two haplotype genomes using Hi-C Integrated Assembly model of hifiasm in order to get its X and Y chromosomes separately, however, juicer pipelines had a bad result, the mapping is very bad, I don't know what the problem is, do you have any suggestions? Or does hifiasm have any parameters to optimize for this?
![image](https://github.com/chhylp123/hifiasm/assets/76806955/c60502cc-c17f-46fa-b611-28d7738ddc99)

Thank you,
Kunjing

**Comment by chhylp123** [comment]
By bad, do you mean that there is no signal?

**Comment by qukunjing** [comment]
> By bad, do you mean that there is no signal?

yes, in fact, there should be 20 chromosomes here, but only nineteen chromosomes are present to signal in the diagram。Can you give me some advices on this situation?
